packages/bitcore-node/src/routes/api/wallet.ts

```diff
@@ -155,6 +159,11 @@ router.post('/:pubKey', authenticate, async (req: AuthenticatedRequest, res) =>
   let keepAlive;
   try {
     let addresses = addressLines.map(({ address }) => address);
+    for (const address of addresses) {
+      if (!Validation.validateAddress(chain, network, address)) {
```

Comment:
This add same vulnerability which insight-api have... but in this time `body-parser` have limit 100mb instead 100kb by default.

Reply:
Which vulnerability? This should prevent large addresses from being created. You could still send 100MB of addresses.

Reply:
But this will not prevent making service very slow. Address validation with 100k symbols length takes ~10s and by default it's possible sent up to 100MB, how much such validation takes?

There should be something like:
```js
const isValid = address.length < 1000 && Validation.validateAddress(chain, network, address)
if (!isValid) {
```

But probably better add length validation to `bitcore-lib` itself.